specs.json

```diff
@@ -128,6 +136,8 @@
   "https://wicg.github.io/client-hints-infrastructure/",
   "https://wicg.github.io/close-watcher/",
   "https://wicg.github.io/compression/",
+  "https://wicg.github.io/compute-pressure/",
+  "https://wicg.github.io/conditional-focus/",
```

Comment:
As simultaneously suggested in https://github.com/w3c/browser-specs/issues/386#issuecomment-927698905, I think this one could better live in monitoring.